src/staggered_nonhydrostatic_model.jl

```diff
@@ -28,12 +28,30 @@ import ClimaCore.Fields: ColumnField
 function precomputed_quantities(Y, atmos)
     FT = eltype(Y)
     TST = thermo_state_type(atmos.moisture_model, FT)
+    precomputed_sgs_quantities = if atmos.turbconv_model isa EDMFX
+        n = n_mass_flux_subdomains(atmos.turbconv_model)
+        (;
+            ᶜuʲs = similar(Y.c, NTuple{n, Geometry.Covariant123Vector{FT}}),
```

Comment:
Would it be useful to add a short comment here explaining what each of those variables is?

Reply:
Sure, will do.